src/test/java/teammates/test/cases/util/FieldValidatorTest.java

```diff
@@ -28,7 +29,8 @@ public void testGetValidityInfoForSizeCappedNonEmptyString() {
         int typicalLength = 25;
 
         try {
-            validator.getValidityInfoForSizeCappedNonEmptyString(typicalFieldName, typicalLength, null);
+            teammates.test.driver.FieldValidator.getValidityInfoForSizeCappedNonEmptyString(typicalFieldName,
```

Comment:
Why not import `teammates.test.driver.FieldValidator` and remove the import for `teammates.common.util.FieldValidator`? Then all these lines don't have to be changed.